### PR TITLE
Issue/112 network map

### PR DESCRIPTION
**Pull request by willgarrett64**
Updated network map.

If the background map image is ever changed, it needs to be Mercator projection, and then `mapLngLeft`, `mapLngRight` and `mapLatBottom` need to be tweaked to fit the new map boundaries. Also, the height/width styles might need tweaking (as well as that of the Statistics component) to keep it all lined up neatly. 

The merge conflict I imagine is just because this branch still had a map on testnet, but that has since been removed and re-replaced with the Faucet.  

image

### CODE REVIEW

File: src/components/NetworkMap.vue [hunk 1]
```diff
@@ -1,118 +1,114 @@
...
+      const mapOffsetY = (worldMapWidth / 2 * Math.log((1 + Math.sin(mapLatBottomRad)) / (1 - Math.sin(mapLatBottomRad))))
 
-      // focus on country if viewing single node
-      const options = {...this.options, height: this.height}
-      if (this.session) options.region = this.session.node.geo.countryCode
+      const x = (lng - this.mapLngLeft) * (this.mapWidth / mapLngDelta)
+      const y = this.mapHeight - ((worldMapWidth / 2 * Math.log((1 + Math.sin(latitudeRad)) / (1 - Math.sin(latitudeRad)))) - mapOffsetY)
 
-      this.map.draw(data, options)
+      return {x, y}
```
Comment (adamkdean) [inline src/components/NetworkMap.vue:49]: Usually better to put spaces in objects, `{ x, y }` etc

File: src/components/NetworkMap.vue [hunk 2]
```diff
@@ -1,118 +1,114 @@
...
+    async drawPoints() {
+      const canvas = this.$refs.mapCanvas
+      const ctx = canvas.getContext('2d')
+      ctx.canvas.width = this.mapWidth
+      ctx.canvas.height = this.mapHeight
+      ctx.fillStyle = '#0ecc5f'
+      ctx.strokeStyle = '#5cbd64'
+
+      this.points.forEach(p => {
+        let {x, y} = this.convertGeoToXy(p.lat, p.lng)
```
Comment (adamkdean) [inline src/components/NetworkMap.vue:60]: Same here `{ x, y }`

File: src/views/Overview.vue [hunk 3]
```diff
@@ -29,8 +29,11 @@ import RecentTransactions from "@/components/RecentTransactions"
 import Statistics from "@/components/Statistics"
 import SummaryHero from "@/components/SummaryHero"
 
+import superagent from 'superagent'
 import { fetchBlocks, fetchStakeStats, fetchTransactions } from '../utils/api'
 
+const mapRefreshInterval = 60 * 1000
```
Comment (adamkdean) [inline src/views/Overview.vue:35]: Any reason for this not being in the data like `pollInterval`?

**Review by adamkdean — APPROVED** [review]
LGTM, few little tweaks but looks good!

File: src/views/Overview.vue (hunk 3 above)
Comment (willgarrett64) [reply src/views/Overview.vue:35]: In a previous component that I did this is where Anny asked me to put the polling/refresh intervals. For consistency, I guess I could move the poll interval out of `data`, and go through Wallet and try to do that throughout. There is still a bit of a mix of approaches

**Pull request merged**